1.9/usage/debugging/cli-debugging/task-exec/index.md

```diff
@@ -1,26 +1,26 @@
 ---
-post_title: Debugging
-feature_maturity: preview
-menu_order: 3.3
+post_title: dcos task exec
+feature_maturity: experimental
```

Comment:
Mark all of these debugging topics as `preview` - https://docs.google.com/spreadsheets/d/12iqPsm7DdiZWfn19x7Z-0tb4H1Se41uP-TclJ4joGko/edit#gid=1924292636